src/start/PasswordAuthView.js

```diff
@@ -109,6 +112,14 @@ class PasswordAuthView extends React.Component {
           onPress={this.validateForm}
         />
         <ErrorMsg error={error} />
+        <Touchable>
```

Comment:
You can also add an item to messages_en.json for the new strings (and skip the other translations as they are generated by Transifex)